app/models/operatingsystem.rb

```diff
@@ -170,7 +170,11 @@ def self.find_by_to_label(str)
     os = find_by_description(str.to_s)
     return os if os
     a = str.split(" ")
-    b = a[1].split('.') if a[1]
+    if a[0] =~ /ubuntu/i
+      b = a[1], nil if a[1]
```

Comment:
Some tests for this would be nice. I'd be worried about `22.04.2` because Ubuntu does have minor releases.

Reply:
Ok, I think I was a little bit to hasty with my changes... Indeed at place if there is a minor version present in the string, I should take it into account :sweat_smile: